<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>成功始于沟通</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="7,0">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="placeholderText">
         <string>暂无消息记录...</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QFontComboBox" name="fontComboBox">
          <property name="styleSheet">
           <string notr="true">QComboBox {
	background-color: rgb(255, 255, 255);
    border: 1px solid gray;
    border-radius: 3px;
    padding: 1px 2px 1px 2px;  /*针对于组合框中的文本内容*/
    /*min-width: 9em;   组合框的最小宽度*/
}
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;

    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid; 
    border-top-right-radius: 3px; 
    border-bottom-right-radius: 3px;
}
QComboBox::down-arrow {
    image: url(:/images/QComboBox);
}
QComboBox::drop-down:hover { background-color: rgb(50, 50, 50,255);}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>9</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>10</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>11</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>12</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>14</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>16</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>18</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>20</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>24</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>28</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>32</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="placeholderText">
         <string>请输入要发送的内容！</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btn_chartClose">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>103</width>
            <height>41</height>
           </size>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::NoContextMenu</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton { background-color: rgb(50, 50, 50,200); border-radius: 3px; color: rgb(255, 255, 255); }
 QPushButton:hover { background-color: rgb(50, 50, 50,255);}
</string>
          </property>
          <property name="text">
           <string>关闭</string>
          </property>
          <property name="shortcut">
           <string>Esc</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="btn_chartSend">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>103</width>
              <height>41</height>
             </size>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::NoContextMenu</enum>
            </property>
            <property name="toolTip">
             <string>按Ctrl+Enter发送消息</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton { background-color: rgb(50, 50, 50,200); border-radius: 3px; color: rgb(255, 255, 255); }
 QPushButton:hover { background-color: rgb(50, 50, 50,255);}
</string>
            </property>
            <property name="text">
             <string>发送</string>
            </property>
            <property name="shortcut">
             <string>Ctrl+Return</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tBtn_setSendBtnShortcut">
            <property name="minimumSize">
             <size>
              <width>23</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>23</width>
              <height>41</height>
             </size>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::ActionsContextMenu</enum>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="arrowType">
             <enum>Qt::DownArrow</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget">
      <item>
       <property name="text">
        <string>好友列表</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>773</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
